portfolio/src/main/webapp/script.js

```diff
@@ -31,12 +31,12 @@ function addRandomFact() {
  * Fetch the list of comments and add them to the page
  */
 function getComments(){
```

Comment:
refrain from using "getBlah()" if the function does not return Blah. Try to use common programming patterns. 
You have two options, either:
1. Split getComments() into 2 functions, getComments() and displayComments()
2. Change the name of getComments() to fetchAndAddComments();